sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/protocol/datalake_rest_client.hpp

```diff
@@ -4,15 +4,14 @@
 
 #pragma once
 
-#include "azure/core/datetime.hpp"
-#include "azure/core/http/http.hpp"
-#include "azure/core/http/pipeline.hpp"
-#include "azure/core/nullable.hpp"
-#include "azure/core/response.hpp"
-#include "azure/storage/common/crypt.hpp"
-#include "azure/storage/common/storage_common.hpp"
-#include "azure/storage/common/storage_exception.hpp"
-
+#include <azure/core/datetime.hpp>
+#include <azure/core/http/http.hpp>
+#include <azure/core/http/pipeline.hpp>
+#include <azure/core/nullable.hpp>
+#include <azure/core/response.hpp>
+#include <azure/storage/common/crypt.hpp>
+#include <azure/storage/common/storage_common.hpp>
+#include <azure/storage/common/storage_exception.hpp>
```

Comment:
I strongly suggest we divide different headers into groups.

I think https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes is a good reference.